server/routes/induction/update/additionalTrainingUpdateController.ts

```diff
@@ -51,6 +56,11 @@ export default class AdditionalTrainingUpdateController extends AdditionalTraini
     }
 
     const updatedInduction = this.updatedInductionDtoWithAdditionalTraining(inductionDto, additionalTrainingForm)
+
+    if (req.session.updateInductionQuestionSet) {
+      // TODO redirect to next page in the question set
```

Comment:
This is as far as I've got (along with the branch to add a qualification). We need to determine the next page here.